### DISCUSSION TITLE
역시 수도권은 많이 힘드네요

### DISCUSSION THREAD

**Discussion by Min0317** [changdoc/naver-vaccine-macro#206] [Q&A]
오늘 김포,강화,인천 3개 섞어서 네이버 지도로 시도했는데 거진 10번의 시도 모두 실패했습니다.  정상적으로 예약까지 진행하는거 보니, 프로그램 자체는 문제없어 보입니다. 

그런데, 동작하고 30분-1시간이 지나면 새로고침 시간이 밀리고 개발자도구 콘솔에 오류가 표출됩니다 (크롬,웨일,서브컴 시도시에도 동일) 저와 같은 오류를 겪는 분이 2분 더 있는걸로 보아 개발자님께서 시간나시면 분석해보시는거도 좋을 듯 합니다.

감사합니다.

**Comment by 1q2w3e4r-1** [comment]
전에 해당 프로그램 말고 다른걸로 네이버로 했을때 새로고침을 너무 누르면 네이버쪽에서 그런건지 업데이트 시간이 밀리더라고요
쿠키 삭제하고 하면 다시 정상으로 돌아왔는데 그때마다 인증서를 재발급해야했어서 카카오로 했습니다

**Comment by changdoc** [comment]
네 어제 테스트 진행하려고 돌려봤으나 잔여 백신이 발생할때까지 문제 없이 동작하여 아직 원인 파악을 못하는 상황이네요.
